app/src/main/java/edu/grinnell/grinnell_publications_android/Services/Templates/JsonPublication.java

```diff
@@ -0,0 +1,34 @@
+package edu.grinnell.grinnell_publications_android.Services.Templates;
+
+/**
+ * Created by dennis on 2/19/17.
```

Comment:
Documentation should go above class not within imports. Also, it should be more descriptive